desktop/app/lib/window-manager/index.js

```diff
@@ -1,13 +1,7 @@
-/**
- * External Dependencies
- */
+/* eslint-disable import/order */
```

Comment:
It seems like the order of `require`s might be relevant which is why I disabled the rule for this file.